src/index.html

```diff
@@ -8,38 +8,40 @@
 Please keep the two in sync when making changes, pending a better solution via git submodules
 -->
 
-  <head>
-    <meta charset="utf-8" />
-    <title>Dockstore</title>
-    <base href="/" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
-    <link rel="manifest" href="manifest.json" />
-    <meta name="theme-color" content="#21335B" />
-    <script type="application/ld+json">
-      {
-        "@context": "http://schema.org",
-        "@type": "Organization",
-        "url": "https://dockstore.org",
-        "logo": "https://raw.githubusercontent.com/dockstore/dockstore-ui2/develop/src/assets/images/dockstore/dockstore.png",
-        "sameAs": ["https://twitter.com/DockstoreOrg"]
-      }
-    </script>
-    <script type="application/ld+json">
-      {
-        "@context": "http://schema.org",
-        "@type": "WebSite",
-        "url": "https://dockstore.org",
-        "sameAs": ["https://twitter.com/DockstoreOrg"],
-        "potentialAction": {
-          "@type": "SearchAction",
-          "target": "https://dockstore.org/search?search={search_term_string}",
-          "query-input": "required name=search_term_string"
-        }
-      }
-    </script>
-  </head>
+<head>
+  <meta charset="utf-8">
+  <title>Dockstore</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="manifest" href="manifest.json">
+  <meta name="theme-color" content="#21335B"/>
+  <script type="application/ld+json">
+  {
+  "@context": "http://schema.org",
+  "@type": "Organization",
+  "url": "https://dockstore.org",
+  "logo": "https://raw.githubusercontent.com/dockstore/dockstore-ui2/develop/src/assets/images/dockstore/dockstore.png",
+  "sameAs": ["https://twitter.com/DockstoreOrg"]
+  }
+  </script>
+  <script type="application/ld+json">
+  {
+  "@context": "http://schema.org",
+  "@type": "WebSite",
+  "url": "https://dockstore.org",
+  "sameAs": ["https://twitter.com/DockstoreOrg"],
+  "potentialAction": {
+    "@type": "SearchAction",
+    "target": "https://dockstore.org/search?search={search_term_string}",
+    "query-input": "required name=search_term_string"
+  }
```

Comment:
odd formatting, did prettier do this?

Reply:
Looks like some changes were made by prettier and some are from one of Shreya's past commits. Should I change back the formatting to the wider indents?

Reply:
what it looks previously is better in my opinion.  if prettier is forcing it to be like it is right now, I'd just leave it as it is and move on for now.